### PR TITLE
Made the program easier to use for people without computer experience

### PR DESCRIPTION
**Pull request by HoboJoe2**
Hi,
I understand if you don't want to merge this request, because I have changed a lot about the program.
I wanted to make it easier for people who don't know anything about programming to use, so I made it so that the program runs from a batch file and has interactive inputs instead of command line arguments.
I also made it so that the program is in verbose mode always.
Thank you for making such an awesome program, I've been listening to random songs while coding this ^_^

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
-This project contains Python files that will play a random song from Spotify by looking up the top thirty songs for a random three-letter string or commonly used word and picking one of them.
+## How to use:
 
-In order to allow the program to authenticate with Spotify, you will need to install the `spotipy` module for Python. You can install the latest version with the command:
+This project allows you to play a random song from Spotify by looking up the top thirty songs for a random three-letter string (char schema) or commonly used word and picking one of them (word schema).
 
-```
-pip install git+https://github.com/plamere/spotipy.git --upgrade
-```
+To run the program, double click on `RandomSpotifySong.bat`. There is also a shortcut file which can be moved from the original folder and runs the same as `RandomSpotifySong.bat`.
```
Comment (parthsarin) [inline README.md]: This is great! `.bat` only work in DOS-based OS systems. Could you add instructions for non-DOS systems for executing the Python file directly?

File: README.md [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
 ```Python
 player = RandomPlayer(random_song_schema='word') # random_song_schema is either 'word' or 'char' depending on how you want to generate random songs (see above)
 player.authenticate()
-
 player.playRandomSong(numSongs = 10) # Will throw an AuthenticationError if player isn't authenticated
 ```
 
 That code would start playback of 10 random songs on Spotify.
 
-You can also do the same thing by executing the following command:
```
Comment (parthsarin) [inline README.md:24]: I think it'd be useful to keep some instructions about running the file with the `python` executable just to keep it OS-independent. Thoughts?

File: RandomSpotifySong.bat [hunk 3]
```diff
@@ -0,0 +1 @@
+python -m venv .env && cd .env/Scripts && activate && cd .. && cd .. && pip install -r requirements.txt && python randomsong.py
```
Comment (parthsarin) [inline RandomSpotifySong.bat]: Hahaha, nice.

File: randomsong.py [hunk 4]
```diff
@@ -164,17 +165,22 @@ def getRandomSongFromChars(self, verbose=False):
 				print("There are no valid songs. Trying again.")
 
 if __name__ == '__main__':
-	if '-v' in sys.argv:
-		verbose = True
+	
+	os.system("cls")
```
Comment (parthsarin) [inline randomsong.py]: Oh this makes the output so much nicer! Could you make this system independent also? The command for linux and bash is `clear` and you should be able to use `os.name` to determine which command to use.

**Review by parthsarin — CHANGES_REQUESTED** [review]
I love what you've done with this! Using verbose mode, clearing the terminal output, etc. are all fantastic changes! Could you make a few of your changes platform independent (see my comments)? Also, could you remove the `Shortcut.lnk` file if it's not necessary?

I appreciate your attention; this should definitely make it easier for non-programmers to use!

**Comment by HoboJoe2** [comment]
Ok, I just pushed the commits to my fork, I'm pretty sure they should show up. This is my first time contributing to a project and using pull requests so I'm still learning.
I don't know much about operating systems other than windows so thanks for the hints about how to make it works on other operating systems.
Thanks for the great feedback, like I said this is my first time contributing to an opensource project and you have been really nice :)

**Comment by parthsarin** [comment]
It looks great, @HoboJoe2! Thanks for helping contribute!

**Pull request merged**